apps/nextra/pages/en/build/smart-contracts/aptos-standards/fungible-asset.mdx

```diff
@@ -1,689 +1,310 @@
-    /// Name of the fungible metadata, i.e., "USDT".
-    name: String,
-    /// Symbol of the fungible metadata, usually a shorter version of the name.
-    /// For example, Singapore Dollar is SGD.
-    symbol: String,
-    /// Number of decimals used for display purposes.
-    /// For example, if `decimals` equals `2`, a balance of `505` coins should
-    /// be displayed to a user as `5.05` (`505 / 10 ** 2`).
-    decimals: u8,
-  }
-}
-```
+The Aptos Fungible Asset Standard (also known as “Fungible Asset” or “FA”) provides a standard, type-safe way to define fungible assets in the Move ecosystem. It is a modern replacement for the `coin` module that allows for seamless minting, transfer, and customization of fungible assets for any use case. 
 
-### Fungible Asset and Fungible Store
+This standard is important because it allows fungible assets on Aptos (such as Currencies and Real World Assets (RWAs)) to represent and transfer ownership in a consistent way dApps can recognize. This standard also allows for more extensive customization than the `coin` module did by leveraging [Move Objects](aptos-object.mdx) to represent fungible asset data. 
 
-FA allows typing by allocating an object reference that points to the metadata.
-Hence, a set of units of FA is represented as an amount and a reference to the
-metadata, as shown:
+The FA standard provides all the functionality you need to create, mint, transfer, and burn fungible assets (as well as automatically allowing recipients of the fungible asset to store and manage any fungible assets they receive). 
 
-```move
-module 0x1::fungible_asset {
-  struct FungibleAsset {
-    metadata: Object<Metadata>,
-    amount: u64,
-  }
-}
-```
+It does so by using two Move Objects:
 
-The FAs is a struct representing the type and the amount of units held. As the
-struct does not have either key or store abilities, it can only be passed from
-one function to another but must be consumed by the end of a transaction.
-Specifically, it must be deposited back into a fungible store at the end of the
-transaction, which is defined as:
-
-```move
-module 0x1::fungible_asset {
-  #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
-  struct FungibleStore has key {
-    /// The address of the base metadata object.
-    metadata: Object<Metadata>,
-    /// The balance of the fungible metadata.
-    balance: u64,
-    /// FAs transferring is a common operation, this allows for freezing/unfreezing accounts.
-    frozen: bool,
-  }
-}
-```
+1. `Object<Metadata>` - This represents details about the fungible asset itself, including information such as the `name`, `symbol`, and `decimals`.
+2. `Object<FungibleStore>` - This stores a count of fungible asset units owned by this account. Fungible assets are interchangeable with any other fungible asset that has the same metadata. An account *may* own more than one `FungibleStore` for a single Fungible Asset, but that is only for advanced use cases.
```

Comment:
I really wish we had a diagram to accompany this. Maybe we could use excalidraw?